apps/angular/src/app/features/anime-table/anime-table.component.html

```diff
@@ -1,25 +1,36 @@
-<table mat-table [dataSource]="(anime$ | async) ?? []" class="anime-table">
+<table
+	mat-table
+	[dataSource]="(animeList$ | async) ?? []"
+	class="anime-table"
+	matSort
+	matSortDisableClear
+	(matSortChange)="onSortChange($event)"
+>
 	@for (column of displayedColumns; track $index) {
-		<ng-container [matColumnDef]="column.key">
-			<th mat-header-cell *matHeaderCellDef class="mat-header-cell">{{column.header}}</th>
-			<td mat-cell *matCellDef="let element" class="mat-cell">
-				@switch (column.key) {
-					@case (columnKey.Image) {
-						<img [src]="element.image" class="table-image" [alt]="element.titleJapanese"/>
-					}
-					@case (columnKey.AiredStart) {
-						{{element.aired.start | date:dateFormat | empty}}
-					}
-					@case (columnKey.TitleEnglish) {
-						{{element.titleEnglish | empty}}
-					}
-					@default {
-						{{element[column.key]}}
-					}
+	<ng-container [matColumnDef]="column.key">
+		@if (sortableFields.includes(column.key)) {
+		<th mat-header-cell *matHeaderCellDef mat-sort-header class="mat-header-cell">{{ column.header }}</th>
+		} @else {
+		<th mat-header-cell *matHeaderCellDef class="mat-header-cell">{{ column.header }}</th>
+		}
```

Comment:
Maybe use 'disabled' property of MatSortHeader instead of 'if' statement? (see [docs](https://material.angular.io/components/sort/api#MatSortHeader)).
```suggestion
		<th mat-header-cell *matHeaderCellDef mat-sort-header class="mat-header-cell" [disabled]="!sortableFields.includes(column.key)">{{ column.header }}</th>
```